README.md

```diff
@@ -24,6 +24,130 @@ This repo uses [`Chromatic`](https://www.chromatic.com/) to streamline reviews b
 
 Our web components have linting which checks for hard-coded user-facing strings. At the moment this linting isn't integrated into CI - so you will only see it if you run `yarn lint` or if your editor has ESLint integration through a plugin.
 
+### Local testing with Verdaccio
+Contributors are encouraged to test their changes in `vets-website` using [Verdaccio](https://verdaccio.org/). What is Verdaccio? From the website:
+
+> Verdaccio is a simple, zero-config-required local private NPM registry. No need for an entire database just to get started&hellip;
+
+Verdaccio allows contributors to publish a new version of the VA Design System on their local machine, and preview `vets-website` with their changes.
+
+#### Installing Verdaccio
+1. Verdaccio requires [Python3](https://www.python.org/downloads/). Ensure you have Python3 installed by opening a terminal window and typing the following command:
+
+   ```shell
+   which python3 # Should see output like /usr/bin/python3
+   ```
+   
+   You should see an execution path if Python3 is installed. If not, install it and type `which python3` again in a new terminal window.
+1. Install Verdaccio using NPM or Yarn:
+
+   ```shell
+   # Install this version of Verdaccio for compatibility with Node v14.15.0.
```

Comment:
I'm not how/where to make this clarification in your steps but when the Verdaccio release is made, it's important that the Node version is the same between the `component-library` and `vets-website` terminal windows. Their node versions should both be `v14.15.0`.

But things get a little tricky when running the component-library build which requires node `v18.x.x`. So we need to switch node versions frequently using `nvm` or whatever between `v14.15.0` and `v18.x.x` depending on the task. I just wonder if it's worth mentioning that somewhere even as a troubleshooting thing or something.

Reply:
Great call @jamigibbs. I added a `Before you begin` section with NVM installation and Node version instructions, and an item to the troubleshooting section. I also added comments in the shell commands where it would make sense to verify Node version in use.

Reply:
Re-reading this comment, I need to make a few more edits. I'll take another pass and push the commit later today.